### DISCUSSION TITLE
Can Renovate be made to work without an onboarding PR?

### DISCUSSION THREAD

**Discussion by nekochanell** [renovatebot/renovate#15522] [Request Help] [answered]
Hi.

I have a Gitlab organization with hundreds of repos. I want Renovate to run on all of them. My current onboarding config is 
```
onboardingConfig: { // This will be placed into repositories without config
    extends: [
		"config:base",
		":disableDependencyDashboard"
	],
  },
```

but **I'd like to make it so Renovate can run without an onboarding PR. Is that possible?** All the onboarding PR does is add this inside the repo
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":disableDependencyDashboard"
  ]
}
```
although I think it redundant. 

I want all repos to use the same config, which I have in the root of the repository where I run Renovate from. This would save me having to merge hundreds of onboarding PRs.

**Comment by rarkins** [comment, marked as answer]
Sure, configure onboarding=false and requireConfig=false. Make sure all config you want is in the `config.js` of the bot.

**Reply by hemeroc** [reply]
Hi, where would I need to place this when using `Renovate OnPremise`?
I would like to have this as organization wide settings.

**Reply by rarkins** [reply]
These can be configured in a `config.js` file, or env (e.g. RENOVATE_ONBOARDING=false, RENOVATE_REQUIRE_CONFIG=false)

**Reply by cjtim** [reply]
@rarkins So if i want to run with extends this's config.js looks like right? am i correct

```js
 module.exports = {
        requireConfig: "optional",
        onboarding: false,
        extends: [
            "local>test/renovate-config-templates",
            "local>test/renovate-config-templates:scala",
            "local>test/renovate-config-templates:global",
        ]
```

**Reply by rarkins** [reply]
Yes, then it will run on repos without onboarding, and with or without a config file

**Reply by Quanalogy** [reply]
@rarkins 
 Hi, nice that this is possible, but might you explain why you would want to do onboarding?
It seems to me that since you made it a default value you expect it to be run like that and might there be functionality missing or issues arising for setting these settings ?

**Comment by Alexander-Shukaev** [comment]
How to achieve the same but on organization level?  I.e. without touching `config.js`?  Setting 

```json
  "onboarding": false,
  "requireConfig": "optional",
```

in `renovate-config/default.json` does not have any effect...

**Reply by rarkins** [reply]
@Alexander-Shukaev this discussion is already answered, please start your own
